docs/proposals/PROPOSAL_E.md

```diff
@@ -121,21 +126,21 @@ The response is an Index of descriptors:
   "manifests": [
     {
       "mediaType": "application/vnd.oci.image.manifest.v1+json",
+      "artifactType": "example/icecream", // pulled up from manifest
```

Comment:
Would this match the examples above in line 85?
FWIW, I beleive iana recommends `thing/example`, but that gets overly simplified, so I'm also ok with `"application/vnd.example.icecream.v1" example @michaelb990 provided above.